plugins/inputs/eventhub/eventhub.go

```diff
@@ -0,0 +1,382 @@
+package eventhub
+
+// TODO: (optional) Test authentication with AAD TokenProvider environment variables?
```

Comment:
If you can, that would be great, otherwise we can work through it when we have a user who needs this authentication method.

Reply:
Moved to README